Client/core/Graphics/CGraphics.cpp

```diff
@@ -27,7 +27,7 @@ using namespace std;
 template <>
 CGraphics* CSingleton<CGraphics>::m_pSingleton = NULL;
 
-const SColor g_rectEdgePixelsData[] = {
+static const SColor g_rectEdgePixelsData[] = {
```

Comment:
Isn't this in the data section already? Why does it need a static?

Reply:
I wanted to make it constexpr, for that you need static. But, I coudnt, so i left the static there. I'll remove it.

Reply:
> I wanted to make it constexpr, for that you need static.

Nice, I didn't know that!

Reply:
Nah, nvm, im stupid.